modules/installation-azure-finalizing-encryption.adoc

```diff
@@ -41,6 +41,19 @@ $ az role assignment create --role "<privileged_role>" \// <1>
 <1> Specifies an Azure role that has read/write permissions to the disk encryption set. You can use the `Owner` role or a custom role with the necessary permissions.
 <2> Specifies the identity of the cluster resource group.
 +
+. Grant a role assignment to the cluster principal so that it can access the encrypted storage.
+.. Obtain the principal identity by running the following command:
++
+[source,terminal]
+----
+$ PRINCIPAL_ID=$(az identity list --resource-group $CLUSTER_RESOURCE_GROUP '[0].principalId')
```

Comment:
1. The command never works.
```
$ az identity list --resource-group anli4102-t5phs-rg '[0].principalId'
unrecognized arguments: [0].principalId
```
2. Go through the whole page, the newly added section sounds doing the same thing as previous procedure 1 and 2. So I guess we can skip adding this new step, but re-use step 1 and 2.
3. But actually step 1 and 2 have wrong descriptions. In `user-managed encryption` case, there are two resource groups, one is pre-created resource group to save user-managed encryption set, another one is created by installer to save cluster resources. so the step should be like: `getting encryption set object id from user-managed encryption resource group` -> `get cluster identity principal id from cluster resource group` -> `assign cluster principal id permissions to access encryption set scope`

Reply:
@jhixson74 can you take a look at the syntax error here? What change do we need to make to the `az identity list` command?

Reply:
There is no syntax error here as I can easily reproduce this on my machine. More likely a version issue. I will come up with another method to accomplish the same thing, though at some point it will come down to using the az tool or to avoid any sort of problem possibly logging into the azure portal.

Reply:
@jhixson74 @jianlinliu Thank you both, I've updated the PR to 1) avoid the syntax issues Jianlin is having and 2) move the new step to immediately after step 2 so that it makes more sense.

John, can you let me know if I need to rework or reword things to take into account Jianlin's point 3 above?

Reply:
@bscott-rh , I have updated the JIRA bug with the new procedure.

Reply:
I saw that John - that's what I used in my latest commit.